presto-native-execution/presto_cpp/main/common/ConfigReader.cpp

```diff
@@ -36,10 +35,10 @@ std::unordered_map<std::string, std::string> readConfig(
       continue;
     }
 
-    std::vector<std::string> configParts;
-    folly::split('=', line, configParts);
-    VELOX_USER_CHECK_EQ(configParts.size(), 2, "Malformed config: {}", line);
-    properties.emplace(configParts[0], configParts[1]);
+    auto delimiterPos = line.find('=');
```

Comment:
Should we check that delimiterPos != std::string::npos? Perhaps, add a test for invalid config.